cli/command/user/update_test.go

```diff
@@ -122,7 +122,7 @@ func TestProcessGroups(t *testing.T) {
 
 	for _, tc := range testcases {
 		t.Run(tc.name, func(t *testing.T) {
-			gotGroups := tc.updateOpts.processGroups(tc.current)
+			gotGroups, _ := tc.updateOpts.processGroups(tc.current)
```

Comment:
Since `processGroup()`'s signature has changed and it returns errors, some test cases returning those errors would be good.